gulpfile.js

```diff
@@ -0,0 +1,214 @@
+const gulp = require('gulp');
+const replace = require('gulp-replace');
+const rename = require('gulp-rename');
+const logger = require('gulplog');
+const path = require('path');
+const fs = require('mz/fs');
+const del = require('del');
+const mergeStream = require('merge-stream');
+const {forkStream, PolymerProject, addServiceWorker, HtmlSplitter, getOptimizeStreams} = require('polymer-build');
```

Comment:
since we already have the `polymer cli` installed, can't we just.......`exec` `polymer build`?

Reply:
Most of the code is from polymer-cli, but having it here means it can run `build:static` and `build:prpl-server` in parallel (since they won't share the same /build output directory).